mk/image.lds.S

```diff
@@ -30,7 +30,7 @@ SECTIONS {
 
 		*(.text)
 		*(.text.*)
-
+		_text_end = .;
```

Comment:
I am not sure you actually need these _text_end, _rodata_len and _data_end since for all these sections starting address and length are already defined. That is, to get the end of .text you can  use something like this &_text_vma + &_text_len.

Reply:
Ok, I removed this variables